nidirect_breadcrumbs/src/NodeThemesBreadcrumb.php

```diff
@@ -80,8 +80,9 @@ public function applies(RouteMatchInterface $route_match) {
 
     $route_name = $route_match->getRouteName();
 
-    // Full node view.
-    if ($route_name == 'entity.node.canonical') {
+    // Full node view or webform confirmation.
+    if (($route_name == 'entity.node.canonical')
+      || ($route_name == 'entity.node.webform.confirmation')) {
```

Comment:
Catch webform confirmation pages in here, they will be given a breadcrumb trail if they have a theme assigned